src/mesh/index.md

```diff
@@ -218,21 +300,35 @@ every cloud and environment.
 <td class="no-mobile"></td>
 </tr>
 <tr>
-<td>Enterprise Support and SLA</td>
+<td>
+Enterprise Support and SLA
+{% info_tooltip %}
+With Kong Mesh we provide 24/7/265 enterprise support with different SLAs, powered by Kong's global customer success and technical support team across all world regions, and access to a vast network of partners for local language support as well. A must to have for enterprise mission-critical deployments.
+{% endinfo_tooltip %}
+</td>
 <td><i class="fa fa-times"></i></td>
 <td><i class="fa fa-check"></i></td>
 </tr>
 <tr>
-<td>Customer Success Packages</td>
+<td>
+Customer Success Packages
+{% info_tooltip %}
+With Kong Mesh we provide access to our implementation and training programs to accelerate the rollout of a service mesh across every team, and to properly train and educate the organization on how to effectively drive business outcomes with the product.
+{% endinfo_tooltip %}
+</td>
 <td><i class="fa fa-times"></i></td>
 <td><i class="fa fa-check"></i></td>
 </tr>
 <tr>
-<td>Envoy Support</td>
+<td>
+Envoy Support
+{% info_tooltip %}
+With Kong Mesh we provide access to the Envoy contributors at Kong to further expand the capabilities of the underlying data plane proxy technology (Envoy) with features that are not currently available in upstream Envoy, in order to remove road blocks and cater to unique enterprise requirements and use-cases.
```

Comment:
```suggestion
With Kong Mesh, we provide access to the Envoy contributors at Kong to further expand the capabilities of the underlying data plane proxy technology (Envoy) with features that are not currently available in upstream Envoy. This can be used to remove road blocks and cater to unique enterprise requirements and use-cases.
```